test/fuzzy.test.js

```diff
@@ -65,6 +65,21 @@ describe('fuzzy', function(){
     });
   });
   describe('.filter', function(){
+    it('should return list untouched when the pattern is undefined', function() {
+      arr = ['aba', 'c', 'cacb']
+      var result = fuzzy.filter(undefined, arr);
+      expect(result).to.include.members(arr)
```

Comment:
since we expect the array to be untouched, we can do a reference equality comparison.

expect(result).equals(arr)